### DISCUSSION TITLE
Library - Alphabet 'R' index not selecting folders

### DISCUSSION THREAD

**Discussion by markblinkhorn** [rern/rAudio#1784] [Fixed Issues] [answered]
I'm not sure whether this might be related to #1776 or not - When I select 'R' on the alphabet bar on my NAS library only the individual fles that begin with 'R' or 'r' are displayed. Folders that begin with 'R' are omitted. I can select 'Q' or 'S' and scroll into the list of folders that begin with 'R'. Every other letter on the Alphabet bar works as expected (with the exception of 'X', I am unable to test 'X' as I have no files or folders that begin with 'X').
I have 3 rAudio R-Pi boxes dotted about the place, they all do the same.

**Comment by markblinkhorn** [comment]
Finally - after about a month - I spotted (and corrected) the weird typo in the title. I must get these glasses checked...

**Comment by markblinkhorn** [comment]
I had occasion to do a vanilla install of 20260216 (using the R-Pi Imager - sweet!) and took the opportunity to check whether I could see the folders on my NAS that begin with 'R'. It is still a problem, has nobody else spotted this yet? Is it just me?

**Reply by ceejayemm** [reply]
Not an issue for me with 20260216 on a RPi3B+ and RPi3A+

Chris

<img width="1633" height="985" alt="Screenshot_20260221_095222" src="https://github.com/user-attachments/assets/28adcab4-ea47-44ee-b283-73780055c8e5" />

<img width="1633" height="985" alt="Screenshot_20260221_095241" src="https://github.com/user-attachments/assets/4c9baad7-0595-4de8-a6a7-6527f1d28e57" />

**Comment by markblinkhorn** [comment]
Thanks Chris, I'll look for a more local reason.

**Comment by markblinkhorn** [comment]
It has got someting to do with .mpdignore but just what, at present, is too subtle for me to elucidate. I have 2 folders on my NAS that are used by DFSr. The security on these folders prevents MPD from accessing them and, since there is nothing useful to MPD in there anyway, I exclude them using .mpdignore. However, unbeknownst to me, the entry for __DFSR_DIAGNOSTICS_TEST_FOLDER__ in .mpdignore was erased sometime ago. Re-insert the entry, I can navigate to folders starting with R using the index button 'R', remove the entry, I can't. (I select 'S' and scroll up.) Go figure.

I have more questions than I have answers right now, but pop back in a little while and I might, just might, have it figured out.

**Comment by markblinkhorn** [comment, marked as answer]
I still don't fully understand the 'why' of this but, if you manually manipulate the contents of .mpdignore then you must update the mpd library twice. Seems counter-intuitive but it does work. I found the easiest way to do this was using the CLI:
`mpc update --wait`
`mpc update --wait`
but I guess you could also use the Library Update feature in the GUI. I'm going to close this now as it is probably only for my own esoteric interest anyway. Thanks for watching :-)
